README.md

```diff
@@ -10,20 +10,20 @@ Content to be unblocked from Cyberoam
 ### 2. Medium
 - **URLs**: https://medium.com/
 - **Blocked Under**: Entertainment
-- **Wikipedia**: The platform is an example of social journalism, having a hybrid collection of amateur and professional people and publications, or exclusive blogs or publishers on Medium,[4] and is regularly regarded as a blog host.
+- **Wikipedia**: The platform is an example of social journalism, having a hybrid collection of amateur and professional people and publications, or exclusive blogs or publishers on Medium, and is regularly regarded as a blog host.
 - **About**: From the last few years, Medium has been on a steady rise and has taken a key position as the canonical medium for posting articles and making blog posts, especially on technology. Because of this, most key technology publications have shifted to using Medium and because it's blocked, it has become increasingly difficult for students to access information there.
 
 ### 3.Quora CDN
 - **URL**:https://www.quora.com/
-- **About**: Quora is a question-answer website where people can get answers and opinions on the questions they ask to the community.Its a great platform where the society interacts and solves its problems.A lot of writers on it are masters of their domain including Prof.Thomas Cormen,Steve Wozniak,Jimmy Wales etc.Their answers on various questions are very inspiring and a treat to read and follow.
+- **About**: Quora is a question-answer website where people can get answers and opinions on the questions they ask to the community. Its a great platform where the society interacts and solves its problems.A lot of writers on it are masters of their domain including Prof.Thomas Cormen, Steve Wozniak, Jimmy Wales etc.Their answers on various questions are very inspiring and a treat to read and follow.
```

Comment:
1. `s/Its/It's`
2. A space before "A lot of writers".
3. A space after "Prof."
4. A space before "Their answers".

Reply:
@ryzokuken On it.Adding changes.

Reply:
@sidntrivedi012 You could either add them to this one or @tonystark97 will.

Reply:
@sidntrivedi012 please make a PR for the same, I'll review and merge them.

Reply:
@ryzokuken Ok.